pygmt/clib/loading.py

```diff
@@ -43,8 +43,8 @@ def load_libgmt():
             error = err
     if error:
         raise GMTCLibNotFoundError(
-            "Error loading the GMT shared library '{}':".format(
-                ", ".join(lib_fullnames)
+            "Error loading the GMT shared library '{}'.\n {}.".format(
+                ", ".join(lib_fullnames), error
```

Comment:
```suggestion
            "Error loading the GMT shared library "
            f"'{', '.join(lib_fullnames)}'.\n {error}."
```
Use an f-string? Not sure if it looks cleaner though :sweat_smile: